cmd/chaos-controller-manager/main.go

```diff
@@ -22,6 +22,9 @@ import (
 	"os"
 	"time"
 
+	grpc_prometheus "github.com/grpc-ecosystem/go-grpc-prometheus"
```

Comment:
```suggestion
	grpcprometheus "github.com/grpc-ecosystem/go-grpc-prometheus"
```

 I think without underscore might be better.


>  When renaming an imported package, the local name should follow the same guidelines as package names (lower case, no under_scores or mixedCaps).

ref: https://go.dev/blog/package-names

Reply:
Ok, I will change it.